mod.yaml

```diff
@@ -74,6 +74,7 @@ Rules:
 	./mods/ra2/rules/aircraft.yaml
 	./mods/ra2/rules/civilian-props.yaml
 	./mods/ra2/rules/trees.yaml
+	./mods/ra2/rules/civilian-naval.yaml
```

Comment:
Note to myself: I will need to update here when rebasing https://github.com/OpenRA/ra2/pull/28.